forge-gui/res/cardsfolder/upcoming/magus_lucea_kane.txt

```diff
@@ -0,0 +1,11 @@
+Name:Magus Lucea Kane
+ManaCost:1 G U R
+Types:Legendary Creature Human Tyranid Wizard
+PT:1/1
+T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ At the beginning of combat on your turn, put a +1/+1 counter on target creature.
+SVar:TrigPutCounter:DB$ PutCounter | ValidTgts$ Creature | CounterType$ P1P1 | CounterNum$ 1
+A:AB$ Mana | PrecostDesc$ Psychic Stimulus - | SubAbility$ DelayTrig | Cost$ T | Produced$ C | Amount$ 2 | SpellDescription$ Add {C}{C}. When you next cast a spell with {X} in its mana cost or activate an ability with {X} in its activation cost this turn, copy that spell or ability. You may choose new targets for the copy. (A copy of a permanent spell becomes a token.)
+SVar:DelayTrig:DB$ DelayedTrigger | AILogic$ SpellCopy | Mode$ SpellCast |  Execute$ TrigCopy | ThisTurn$ True | Mode$ SpellCast | ValidCard$ Card.hasXCost | OneOff$ True | ValidActivatingPlayer$ You |
```

Comment:
SpellCast is not for abilities

Reply:
This version is currently not working but is it closes how it should be ?